README.md

```diff
@@ -1,8 +1,21 @@
-# JanusGraph .NET driver
+# JanusGraph.Net
 
-## License
+JanusGraph.Net is the .NET driver of [JanusGraph](http://janusgraph.org/). It is based on Apache TinkerPop™'s [Gremlin.Net](http://tinkerpop.apache.org/docs/current/reference/#gremlin-DotNet).
```

Comment:
If this is "based on Apache TinkerPop's Gremlin.Net`, where is all the code that should be marked as (c) Apache Software Foundation or similar?

All the files in this PR appear to be marked as "(c) JanusGraph.Net authors" of which there are currently only 2 (listed in the `CONTRIBUTORS.txt` file). Note that if any files were copied from Apache TinkerPop, even if they were modified afterwards, must retain the original copyright header. We can then add "portions copyright by JanusGraph.Net authors" if that's appropriate, but we should not be removing existing copyright headers in any imported files.

Reply:
Also, given that this is based on a different piece of software, this PR should also update `LICENSE.txt` to be more similar to the JanusGraph `LICENSE.txt` in that it talks about "all changes prior to date <X> are under the Apache 2.0 license, and all changes beginning with date <X> are under the dual license of Apache 2.0 for code + CC-BY-4.0 for documentation".

See https://github.com/JanusGraph/janusgraph/blob/master/LICENSE.txt for details.

Reply:
What I meant with that sentence is that Gremlin.Net is its core library, not that any code is included directly from Gremlin.Net or TinkerPop in general.
JanusGraph.Net just extends Gremlin.Net with support for types that are JanusGraph specific but the core functionality of the driver is provided by Gremlin.Net. Users should now that fact so that they don't think that they have to decide between Gremlin.Net and JanusGraph.Net as the latter is basically just an extension of Gremlin.Net.
So, I think that the license headers and the `LICENSE.txt` are appropriate.

Do you think that the sentence is misleading?